source/prerequisite-tasks.js

```diff
@@ -13,7 +13,7 @@ module.exports = (input, pkg, options) => {
 		{
 			title: 'Ping npm registry',
 			skip: () => pkg.private || isExternalRegistry,
-			task: async () => npm.checkConnection()
```

Comment:
Don't do unrelated changes. It's true that the `async` keyword is not needed here, but I prefer to add it for readability, as I can immediately know it's an async function.